assets/app/views/services.html

```diff
@@ -7,14 +7,17 @@
         <div>
           <em>{{emptyMessage}}</em>
         </div>
-      </div>      
+      </div>
       <div style="margin-bottom: 10px;" ng-repeat="service in services">
         <h3>{{service.metadata.name}}</h3>
         <div>Created: <relative-timestamp timestamp="service.metadata.creationTimestamp"></relative-timestamp></div>
         <div>IP: {{service.spec.portalIP}}</div>
-        <div>Port: {{service.spec.port}}</div>
-        <div>Container port: {{service.spec.containerPort}}</div>
-        <div>Protocol: {{service.spec.protocol}}</div>
+        <div>Ports:</div>
+        <ul>
+          <li ng-repeat="item in service.spec.ports | orderBy:port">
```

Comment:
Sure, I can make the change. The only reason I hesitated is you'll have "port.port", which I thought would be confusing.